pkg/network/ebpf/c/tracer.h

```diff
@@ -106,7 +107,7 @@ typedef struct {
 
 // Must match the number of conn_t objects embedded in the batch_t struct
 #ifndef CONN_CLOSED_BATCH_SIZE
-#define CONN_CLOSED_BATCH_SIZE 4
+#define CONN_CLOSED_BATCH_SIZE 3
```

Comment:
Stack size limitation?

Reply:
Yes, although I saw @guyarb recently made some space by twiddling with the packing by moving the fields around. I may try that here.

Reply:
I was managed to reduce the space allocation a little bit
by moving `__u8 direction` in `conn_stats_ts_t` to be the last variable, that should give us 3 additional bytes in that struct, and use `__attribute__ ((packed))`, otherwise we will have 3 bytes padding

we can reduce `__u32 metadata` in `conn_tuple_t` as it does not use more than 1 byte

Reply:
rather than that, I think we can use `__attribute__ ((packed))` in all structs to reduce the occupied size